libraries/core_libs/network/src/tarcap/packets_handlers/votes_sync_packet_handler.cpp

```diff
@@ -101,9 +101,9 @@ void VotesSyncPacketHandler::process(const PacketData &packet_data, const std::s
       }
 
       std::vector<std::shared_ptr<Vote>> send_next_votes_bundle;
-      for (auto &v : next_votes) {
+      for (const auto &v : next_votes) {
         if (!peer_to_share_to.second->isVoteKnown(v->getHash())) {
-          send_next_votes_bundle.push_back(std::move(v));
+          send_next_votes_bundle.push_back(v);
```

Comment:
I must be blind. but why the original version didnt work ? I dont see it

Reply:
this loop is inside another one with peers